.github/workflows/basic-ci-amd64.yaml

```diff
@@ -25,7 +23,7 @@ jobs:
       matrix:
         containerd_version: ['lts', 'active']
         vmm: ['clh', 'dragonball', 'qemu', 'stratovirt', 'cloud-hypervisor', 'qemu-runtime-rs']
-    runs-on: garm-ubuntu-2204-smaller
+    runs-on: ubuntu-latest
```

Comment:
I wonder if we'd bee better off picking a pinned version (so `ubuntu-22.04` in this case) that would insulate us from potentially hitting a breakage that is very hard to understand when `ubuntu-24.04` comes out of beta? I know we haven't necessarily followed that approach elsewhere and it might need some effort to try bumping these, but that only happens every couple of years?

Reply:
I agree with @stevenhorsman 's rationale and my preference is to pin to ubuntu-22.04

Reply:
Agreed, I've run into that in other projects. Done!